ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2135,5 +2135,81 @@
         "name": "type-java.svg",
         "mediaType": "image/svg+xml"
     }
+  },
+  {
+    "id": "spring-boot",
+    "creator": "Gytis Trikleris",
+    "name": "Spring Boot",
+    "description": "Spring Boot Stack on CentOS.",
+    "scope": "advanced",
+    "tags": [
+      "Java",
+      "JDK",
+      "Maven",
+      "Spring Boot",
+      "CentOS",
+      "Git"
+    ],
+    "components": [
+      {
+        "name": "CentOS",
+        "version": "---"
+      },
+      {
+        "name": "JDK",
+        "version": "1.8.0_45"
+      },
+      {
+        "name": "Maven",
+        "version": "3.2.2"
```

Comment:
You're installing maven 3.3
https://github.com/eclipse/che-dockerfiles/blob/2fe67f2254109dd2ed649042443998b2799af9e1/recipes/centos_spring_boot/Dockerfile#L18